spec/factories/activity.rb

```diff
@@ -42,6 +42,8 @@
       accountable_organisation_name { "Department for Business, Energy and Industrial Strategy" }
       accountable_organisation_reference { "GB-GOV-13" }
       accountable_organisation_type { "10" }
+
+      association :extending_organisation, factory: :beis_organisation
```

Comment:
I am not sure this should be here?

Reply:
Me either! Where did that come from??

Reply:
Aha seems the XML isn't valid without it, but I will add it as a separate commit because it doesn't belong with this test refactor